src/core/Group.cpp

```diff
@@ -187,6 +187,32 @@ QString Group::defaultAutoTypeSequence() const
     return m_data.defaultAutoTypeSequence;
 }
 
+QString Group::effectiveAutoTypeSequence() const
+{
+    QString sequence;
+    bool enableSet = false;
+
+    const Group* group = this;
+    do {
+        if (!enableSet) {
```

Comment:
This is the code that directly come from `src/autotype/Autotype.cpp` -> `AutoType::autoTypeSequence`
I will do a cleanup (maybe also in the autoTypeSequence function?)